apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -626,7 +630,7 @@ export default Vue.extend({
 
         // callbacks
         ajaxRequestFunc: this.dataFetch,
-        index: this.primaryKey,
+        index: this.primaryKeys,
```

Comment:
I don't think this will work? You might need to make a new hidden column which is a combination of the two and use that for indexing, see http://tabulator.info/docs/5.1/data#row-index